RUFAS/biophysical/animal/pen.py

```diff
@@ -18,54 +26,98 @@
 from RUFAS.data_structures.pen_manure_data import PenManureData
 from RUFAS.data_structures.feed_storage_to_animal_connection import RUFAS_ID, Feed
 from RUFAS.enums import AnimalCombination
+from RUFAS.output_manager import OutputManager
 
 
 class Pen:
     """
     This class represents a pen that houses animals during the simulation.
 
+    Parameters
+    ----------
+    pen_id : int
+        Unique identifier for the pen.
+    pen_name : str
+        Name of the pen.
+    vertical_dist_to_milking_parlor : float
+        Vertical distance from the pen to the milking parlor (m).
+    horizontal_dist_to_milking_parlor : float
+        Horizontal distance from the pen to the milking parlor (m).
+    number_of_stalls : int
+        Number of stalls available in the pen.
+    housing_type : str
+        Type of housing used in the pen.
+    bedding_type : str
+        Type of bedding material used in the pen.
+    pen_type : str
+        The pen type (e.g., lactating, dry, etc.).
+    manure_handling : str
+        Method of manure handling associated with the pen.
+    manure_separator : str
+        Type of manure separator applied in the pen.
+    manure_separator_after_digestion : str
+        Additional manure separation methods used after digestion.
+    manure_storage : str
+        Storage method for manure from the pen.
+    animal_combination : AnimalCombination
+        Combination of animal categories housed in the pen.
+    max_stocking_density : float
+        Maximum allowable stocking density for animals in the pen.
+    minutes_away_for_milking : int
+        Time required to reach the milking parlor from the pen (in minutes).
+    first_parlor_stream : str | None
+        Name of the processor to which the parlor stream will be sent.
+    parlor_stream_name : str | None
+        Name of the parlor stream.
+    manure_streams : list[dict[str, str | float]]
+        List of dictionaries containing manure stream information.
+
     Attributes
     ----------
     id : int
         Internal identifier for the pen.
     pen_name : str
         Name of the pen.
-    max_stocking_density : float
-        Maximum allowable stocking density for animals in the pen.
     vertical_dist_to_parlor : float
         Vertical distance from the pen to the milking parlor, in meters.
     horizontal_dist_to_parlor : float
         Horizontal distance from the pen to the milking parlor, in meters.
     num_stalls : int
         Total number of stalls available in the pen.
     housing_type : str
-        Type of housing in the pen.
+        Type of housing used in the pen.
     bedding_type : str
         Type of bedding material used in the pen.
     pen_type : str
-        The pen type.
+        The pen type (e.g., lactating, dry, etc.).
```

Comment:
```suggestion
        The pen type (freestall, tiestall, open lot, or bedded pack).
```
Per the default metadata properties